include/sway/tree/arrange.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _SWAY_ARRANGE_H
+#define _SWAY_ARRANGE_H
+
+struct sway_container;
+
+void arrange_windows(struct sway_container *container);
+
+// Determine the root container's geometry, then iterate to everything below
+void arrange_root();
```

Comment:
Explicitly put `(void)` for the arguments.
Empty argument lists in prototypes have surprising results in C.